tox.ini

```diff
@@ -29,6 +29,14 @@ commands =
     sphinx-build -W -b latex -d {envtmpdir}/doctrees docs docs/_build/latex
     sphinx-build -W -b doctest -d {envtmpdir}/doctrees docs docs/_build/html
 
+[testenv:mypy]
+basepython = python3
+deps =
+    mypy
+commands =
+    mypy packaging --strict
```

Comment:
nit: Let's have a `mypy.ini` that sets strict to 1/true/whatever, so that my alias of `tox-mypy` (== `./.tox/mypy/bin/mypy`) would do the right thing. :3

Reply:
I actually couldn't figure out how to set the equivalent of `--strict` via the config file without just enabling all the specific flags it enables (which, BTW, may change between mypy versions) but maybe I'm missing something: https://mypy.readthedocs.io/en/latest/config_file.html

Reply:
I guess that's a point in favour of not using --strict. 🙃

Anyway, I don't want to block on this concern. Let's look into this in a follow up?

Reply:
Looks like they didn't provide this on purpose: https://github.com/python/mypy/issues/3858